### PR TITLE
Add pandas-highcharts recipe

### PR DESCRIPTION
**Pull request by ericdill**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+*.swp
```
Comment (jakirkham) [inline .gitignore]: Was this suppose to be included? If so, let's make it a separate PR.

File: recipes/pandas-highcharts/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+{% set version = "0.5.2" %}
+{% set name = "pandas-highcharts" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: pandas-highcharts-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```
Comment (jakirkham) [inline recipes/pandas-highcharts/meta.yaml]: Might want to switch `pypi.python.org` -> `pypi.io.`

File: recipes/pandas-highcharts/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  md5: 7e410ddcdfca6c13c3ad7c741088dc52
```
Comment (jakirkham) [inline recipes/pandas-highcharts/meta.yaml]: There are `sha256` checksums on [pypi.org]( https://pypi.org/ ).

File: .gitignore (hunk 1 above)
Comment (ericdill) [inline .gitignore]: Yes it was supposed to be included. I agree it makes more sense as a separate PR

File: recipes/pandas-highcharts/meta.yaml (hunk 2 above)
Comment (ericdill) [inline recipes/pandas-highcharts/meta.yaml]: done

File: recipes/pandas-highcharts/meta.yaml (hunk 3 above)
Comment (ericdill) [inline recipes/pandas-highcharts/meta.yaml]: good to know. updated the recipe. thanks

**Comment by ericdill** [comment]
@jakirkham cleaned up recipe based on your comments. Thanks for the review.  Should be good to go now

**Comment by ericdill** [comment]
Thanks @ocefpaf !

**Comment by jakirkham** [comment]
Thanks @ericdill. Late to the party, but LGTM.
